source/features/html-preview-link.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {groupSiblings} from '../libs/group-buttons';
+
+function init(): void {
+	const rawButton = select('#raw-url') as HTMLAnchorElement;
+	if (rawButton && /\.html?/.test(rawButton.pathname)) {
```

Comment:
I think you can move this to the feature’s `include`, by creating a `isHTMLSingleFile` test function (not in the current file, not `page-detect.ts`

Reply:
> I’m gonna guess that if (rawbutton) isn’t necessary in that case.

Yeah I doubted that when I first read it but didn't check, come to think it now, there will always be a raw button for files.

Reply:
> I think you can move the regex test to the feature’s include, by creating a isSingleHTMLFile test function (not in the current file, not page-detect.ts

😅 Honestly, I don't know where to put it then, so I put it into `utils.ts`.